docs/serving/autoscaling/autoscaling.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Autoscaling concepts"
+linkTitle: "Autoscaling concepts"
+weight: 10
+type: "docs"
+---
+
+Knative Serving provides users with the functionality to automatically scale applications based on demand.
+
+The component responsible for autoscaling functionality in Knative Serving is the Autoscaler resource. The Autoscaler watches traffic flow to the application, and scales the application up or down based on metrics.
+<!--Question: where does the Autoscaler get metrics from, what format?-->
+<!--Question: what is the relationship between Autoscaler and KPA? Are they the same or does KPA plug into this resource?-->
+The default Autoscaler resource used in Knative deployments is the Knative Pod Autoscaler (KPA). However, Knative deployments can be configured to use the Kubernetes Horizontal Pod Autoscaler (HPA), or a custom third party Autoscaler.
+
+<!--TODO Add some comparison between the three types-->
+Knative Serving revisions are configured by default to use generic settings that are suitable for the majority of use cases. However, some workloads may required a custom, more finely-tuned configuration.
+This guide provides information about configuration options that you can modify to fit the requirements of your workload.
+
+## Autoscaling concepts
+
+The following sections describe important concepts and terms used when discussing autoscaling in Knative.
+
+* [Global versus per revision settings](./global-vs-per-revision.md)
+* [Supported Autoscaler types](./supported-autoscaler-types.md) (includes configuration settings for enabling HPA)
+* [Metrics and targets](./autoscaling-metrics-targets.md)
+* [Concurrency](./concurrency.md)
+* 
```

Comment:
<!--whitespace-bot-->
Remove trailing whitespace:
```suggestion
*
```

Reply:
I wouldn't bother reviewing this until I'm done with a draft, just FYI :smile: